src/pages/components/tile/usage.mdx

```diff
@@ -366,88 +401,95 @@ the chevron icon.
 <Row>
 <Column colLg={8}>
 
-![Tile anatomy](images/tile-usage-6.png)
+![Tile anatomy](images/tile-usage-8.png)
 
 </Column>
 </Row>
 
-1. **Container**: Includes a title, optional label, and the close icon.
+1. **Image (optional)** : The image follows the aspect ratios with the default
+   size of 4:3
+
+2. **Container**: Contains the content area
 
-2. **Content area**: Contains the information and/or controls needed to complete
+3. **Content area**: Contains the information and/or controls needed to complete
    the modal's task. It can include message text and components.
 
 ### Sizing
 
 The **width** varies depending on three basic grid modes: wide, narrow, and
 condensed. The **height** varies depending on the amount of content placed
-within it, but still follows the aspect ratio.
+within it, but applies the
+[spacing scale](https://carbondesignsystem.com/guidelines/spacing/overview/#spacing-scale)
+and follows the
+[aspect ratio](https://www.carbondesignsystem.com/guidelines/2x-grid/overview/#aspect-ratio).
+The minimum height starts with 2:1 aspect ratio. As the content grows, set
+vertical spacing between sections within the content area.
 
-[Aspect ratios](https://www.carbondesignsystem.com/guidelines/2x-grid/overview/#aspect-ratio)
-are written out as a formula of width to height, such as 3:2. It’s important to
-remember that while two images can have the same aspect ratio, they can have
-varying images sizes.
+### Alignment
 
-<Row>
-<Column colLg={8}>
+#### Icon and call-to-action button
 
-![Example image of expandable tiles.](images/tile-usage-7.png)
+On clickable tiles, use the arrow icon with the `interactive` token in light
+themes and the `icon-primary` token for dark themes. Place the icon at the
+bottom right to indicate navigation. Avoid using interactive links or buttons
```

Comment:
Avoid confusing the user with multiple click targets (like interactive links or buttons) on clickable tiles.